test-polyfill.mjs

```diff
@@ -26,6 +26,14 @@ test('standard vectors', async t => {
   }
 });
 
+test('omitPadding', async t => {
+  for (let [string, result] of standardBase64Vectors) {
+    await t.test(JSON.stringify(string), () => {
```

Comment:
nbd, but why is this async? isn't this all sync?

Reply:
I just find it easier to always `await` everything in `node:test` rather than thinking about which things need it.